### DISCUSSION TITLE
I wonder how to get isScrolling in Items of List?

### DISCUSSION THREAD

**Discussion by nonoroazoro** [petyosi/react-virtuoso#414] [Q&A]
Like isSeeking? But it seems I can't use useEmitterValue('isScrolling').

P.S. This came about because I wanna add "pointer-events: none" to the list while scrolling.

<img width="1100" alt="图片" src="https://user-images.githubusercontent.com/383132/126777626-ba2ff45f-177b-41de-9de5-968c7d76543a.png">

**Comment by petyosi** [comment]
I am thinking if exposing those hooks in the item content or in the item component won't be a good idea... or too much power.

**Reply by petyosi** [reply]
Apart from that, there's an event called scrollingStateChange which you can use, but you will need some manual hook work there.

**Reply by nonoroazoro** [reply]
> 
> 
> I am thinking if exposing those hooks in the item content or in the item component won't be a good idea... or too much power.

I just wanna add a style corresponding to the scroll state, so it makes sense to keep the hooks inside the component. I'll try scrollingStateChange, thanks!
